docs/installation.rst

```diff
@@ -39,6 +39,7 @@ Add the following apps to the ``INSTALLED_APPS``:
         'django_otp.plugins.otp_static',
         'django_otp.plugins.otp_totp',
         'two_factor',
+        'two_factor.plugins.phonenumber',  # <- if you want phone number capability.
     )
```

Comment:
Might be nice to have `two_factor.plugins.yubikey` in here too. I know it's not technically needed, but it does advertise the fact that we support yubikey.